library/commandline/test/commandline_test.rb

```diff
@@ -19,18 +19,22 @@
     allow(Yast::Debugger).to receive(:installed?).and_return(false)
   end
 
+  # NOTE: when using the byebug debugger here temporarily comment out
+  # all "expect($stdout)" lines otherwise the byebug output will be
+  # lost in the rspec mocks and you won't see anything.
+
   it "invokes initialize, handler and finish" do
-    expect(STDOUT).to receive(:puts).with("Initialize called").ordered
-    expect(STDOUT).to receive(:puts).with("something").ordered
-    expect(STDOUT).to receive(:puts).with("Finish called").ordered
+    expect($stdout).to receive(:puts).with("Initialize called").ordered
+    expect($stdout).to receive(:puts).with("something").ordered
+    expect($stdout).to receive(:puts).with("Finish called").ordered
 
     Yast::WFM.CallFunction("dummy_cmdline", ["echo", "text=something"])
   end
 
   it "displays errors and aborts" do
-    expect(STDOUT).to receive(:puts).with("Initialize called").ordered
-    expect(Yast::CommandLine).to receive(:Print).with(/I crashed/).ordered
-    expect(STDOUT).to_not receive(:puts).with("Finish called")
+    expect($stdout).to receive(:puts).with("Initialize called").ordered
+    expect(Yast::Report).to receive(:LongError).with(/I crashed/, anything).ordered
+    expect($stdout).to_not receive(:puts).with("Finish called")
```

Comment:
that is strange, how it can work previously? Before that PR we also open Error when program crashes.

Reply:
That's what I would also like to know. Especially when it builds in travis, so may be an inconsistency in build envs?